### DISCUSSION TITLE
iwlwifi info

### DISCUSSION THREAD

**Discussion by kimocoder** [ZerBea/hcxdumptool#365] [General]
iwlwifi captures the PMKID and all... but on aireplay-ng -9 <device> its not responding back on injected frames, even though its sending/receiving correctly.

**Comment by ZerBea** [comment]
Thanks for this additional information.
I've moved it to discussions, because these chipsets (Intel) are problematic.
Unfortunately it looks like monitor mode and frame injection don't have a high priority:
https://bugzilla.kernel.org/show_bug.cgi?id=214291
